cli/command/cli.go

```diff
@@ -232,6 +250,15 @@ func getDiscovery(discoveryFlag string) string {
 	return discoveryHost
 }
 
+func getTimeout() (int, error) {
+	timeoutStr := os.Getenv(cliconfig.EnvStorageOSTimeout)
+
+	if timeoutStr == "" {
+		return cliconfig.DefaultTimeout, nil
+	}
+	return strconv.Atoi(timeoutStr)
```

Comment:
Combined with using a `time.Duration`, we can give the user a nicer experience by accepting different units for free:
```suggestion
	return time.ParseDuration(timeoutStr)
```

This allows the user to specify values like "5s" or "1m" - however if they specify "5" it'll default to 5 nanoseconds, so we should have some logic to convert values without a unit to seconds.

Reply:
done

Reply:
Given this needs a duration unit, I think we should either implement the above (defaulting to seconds if no unit is provided) or include (a subset of) the acceptable units in the help text.

After that, looks good to me.

Reply:
Or maybe just intercepting the error and printing something more helpful? I'll let you see what works best 👍

Reply:
Added some examples in the help text.